include/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth.hpp

```diff
@@ -49,16 +49,34 @@ class Earth : public Model
     {
         Undefined,    ///< Undefined
         Exponential,  ///< Exponential atmospheric density model, valid up to 1000 km
-        NRLMSISE00    ///< Navy Research Lab Mass Spectrometer and Incoherent Scatter Radar Exosphere 2000
+        NRLMSISE00,   ///< Navy Research Lab Mass Spectrometer and Incoherent Scatter Radar Exosphere 2000
+    };
+
+    enum class InputDataSourceType
```

Comment:
```suggestion
    enum class InputDataType
```
nit: I think this could be shortened to either `InputDataType` or `InputDataSource` and still be clear IMO.

Reply:
Yeah 💯

Reply:
applied